pylmod/gradebook.py

```diff
@@ -56,8 +56,10 @@ def get_assignments(self, gradebookid='', simple=False):
                       includeGradingStats='false')
 
         dat = self.get(
-            'assignments/{gradebookId}',
-            params=params, gradebookId=gradebookid or self.gradebookid
+            'assignments/{gradebookId}'.format(
+                gradebookId=gradebookid or self.gradebookid
+            ),
+            params=params,
         )
         if simple:
             return [{'AssignmentName': x['name']} for x in dat['data']]
```

Comment:
It is because it is a list comprehension, and since the variable has no life outside this line, there isn't really a need to be descriptive.  Presumably the thing you are assigning it to, or the variable you are extracting the list from will be descriptive enough.